openedx/core/djangoapps/video_config/tests/test_models.py

```diff
@@ -13,100 +13,196 @@
 
 
 @contextmanager
-def hls_playback_feature_flags(
+def video_feature_flags(
+        all_courses_model_class, course_specific_model_class,
         global_flag, enabled_for_all_courses=False,
         course_id=None, enabled_for_course=False
 ):
     """
-    Yields HLS Playback Configuration records for unit tests
+    Yields video feature configuration records for unit tests
     Arguments:
         global_flag (bool): Specifies whether feature is enabled globally
```

Comment:
Any description for` all_courses_model_class, course_specific_model_class` would be helpful.